build.gradle

```diff
@@ -370,6 +375,11 @@ def configureCluster(OpenSearchCluster cluster, Boolean securityEnabled) {
 integTest {
     useCluster testClusters.leaderCluster
     useCluster testClusters.followCluster
+    if (System.getProperty("tests.rest.bwcsuite") == null) {
+        filter {
```

Comment:
Should we always exclude these as part of integTest as clusters are not available for running bwc tests?